invenio_rdm_migrator/streams/oauth/transform.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM migration record transform interfaces."""
+
+from abc import abstractmethod
+
+from ...transform import EncryptMixin, IdentityDictKeyMixin, Transform
+
+
+class OAuthServerTokenTransform(Transform, EncryptMixin, IdentityDictKeyMixin):
+    """OAuth server token data transformation."""
+
+    SCOPES_MAPPING = {
```

Comment:
For the keys (old invenio v3) I've checked Zenodo DB and also via `current_oauth2server.scopes`. Doing the same in RDM instances (local, zenodo-rdm, inveniordm demo site) only returns one value. I'm leaving it as is for now, when the scopes are implemented/registered it's just a matter of modifying this mapping (or subclassing in the corresponding instance.